apps/nextjs-app/src/features/app/blocks/import-table/field-config-panel/inplace-panel/InplacePreviewColumn.tsx

```diff
@@ -25,7 +25,31 @@ interface IPreviewColumnProps {
   onChange: (columns: IInplaceImportOptionRo['insertConfig']['sourceColumnMap']) => void;
 }
 
-const UNSUPPORTFIELDTYPES = [FieldType.User, FieldType.Rollup, FieldType.Formula];
+// all field support in inplace-import
+const SupportTypeMaps: Record<FieldType, boolean> = {
```

Comment:
During import in existing table, you should use `isComputed` to determine whether the import is allowed, because there may be a lookup field